contracts/LicenseToken.sol

```diff
@@ -102,6 +112,7 @@ contract LicenseToken is ILicenseToken, ERC721EnumerableUpgradeable, AccessManag
             uint256 tokenId = $.totalMintedTokens++;
             $.licenseTokenMetadatas[tokenId] = ltm;
             _mint(receiver, tokenId);
+            _getLicensorStorage().licensorIpTokens[licensorIpId].push(tokenId);
```

Comment:
this could end up being quite gas intensive operation if user mints for example 1000 licenses.

If the goal is to know if an IP has minted a license - couldn't this be a boolean or a counter?